internal/descriptor/cache/key_test.go

```diff
@@ -0,0 +1,93 @@
+package cache_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/internal/descriptor/cache"
+)
+
+func TestGenerateDescriptorCacheKey(t *testing.T) {
+	t.Parallel()
```

Comment:
Does it make sense to run this test suite as a parallel one? It is a lighting fast one, so the context switching for it will take longer than running it sequentially 😁